### ISSUE TITLE
Do you read the docs of Django before coding?

### ISSUE THREAD

**Issue by VunkLai** [LeoTunero/first_side_project#10]
Your code: `title = models.CharField('標題', max_length=20)`

The func API: `class CharField(max_length=None, **options)`
In Django, model.CharField has two arguments (keyword arguments) `max_length` and `options` only.
There are not any positional arguments.

When you coding, type a word, do you known what's that mean?
FYR: [models.CharField](https://docs.djangoproject.com/en/3.2/ref/models/fields/#charfield)

**Comment by VunkLai** [comment]
There is an important question.
Yes, the code is works maybe. But way?

**Comment by LeoTunero** [comment]
According to the last meeting, I thought you told me that my tempo of updating is too slow.
So I thought I should make the program works first.
Afterward, I am going to reconstruct my code step by step.

**Comment by VunkLai** [comment]
I said "too slow", it means you should do more, fix a bug, enhance an issue, create a feature, etc.
I do not say, 'Focus on your project, ignore others'

Some issues are important to you, the others not.
which one is most important, issue or feature?

If an issue apply `invalid`, `bug` or `wontfix` label, it is very important.
An issue has low priority, its label is documentation, question, etc.
The issue with `enhancement` label is equal to a `feature` issue.

Now, do you have an ordered todo-list?
```
invalid / bug / wontfix -> project feature / enhancement -> documentation, question -> ...
```

**Comment by LeoTunero** [comment]
okay 
my fault 
I have fixed the problem with 784fafa

**Comment by VunkLai** [comment]
GJ.
Safe to close this issue.
